Index: src/main
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>mmmmmmmm
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main b/src/main
--- a/src/main	
+++ b/src/main	
@@ -1,1 +1,0 @@
-mmmmmmmm
\ No newline at end of file
Index: src/main
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>mmmmmmmm
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main b/src/main
--- a/src/main	
+++ b/src/main	
@@ -1,1 +1,0 @@
-mmmmmmmm
\ No newline at end of file
