Index: src/Main.java
===================================================================
diff --git a/src/Main.java b/src/test/java/src/Main.java
rename from src/Main.java
rename to src/test/java/src/Main.java
--- a/src/Main.java	
+++ b/src/test/java/src/Main.java	
@@ -1,5 +1,7 @@
 package src;
 
+import src.algorithms.Program;
+
 public class Main {
     public static void main(String args[]){
         Program program = new Program();
