Index: src/Main.java
===================================================================
diff --git a/src/Main.java b/src/Main.java
deleted file mode 100644
--- a/src/Main.java	(revision eef57182dbd121595d23b0afded9b1a7b08d0c6b)
+++ /dev/null	(revision eef57182dbd121595d23b0afded9b1a7b08d0c6b)
@@ -1,8 +0,0 @@
-package src;
-
-public class Main {
-    public static void main(String[] args)
-    {
-
-    }
-}
diff --git a/target/generated-sources/annotations/Main.java b/target/generated-sources/annotations/Main.java
new file mode 100644
