Index: src/FillService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/FillService.java b/src/test/java/src/algorithms/FillService.java
rename from src/FillService.java
rename to src/test/java/src/algorithms/FillService.java
--- a/src/FillService.java	(revision 3aee43697d4282a5d5586c34909be628fe5f1be8)
+++ b/src/test/java/src/algorithms/FillService.java	(date 1743008680033)
@@ -1,13 +1,11 @@
-package src;
+package src.algorithms;
 
 import java.lang.reflect.Field;
-import java.lang.reflect.InvocationTargetException;
 import java.util.*;
-import java.util.regex.Pattern;
 
-import static src.ClassInspector.*;
+import static src.algorithms.ClassInspector.*;
 
-public class FillService implements Service{
+public class FillService implements Service {
     private List<Object> mainCollection = new ArrayList<>();
     private Validator validator = new Validator();
 
Index: src/Validator.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Validator.java b/src/test/java/src/algorithms/Validator.java
rename from src/Validator.java
rename to src/test/java/src/algorithms/Validator.java
--- a/src/Validator.java	(revision 3aee43697d4282a5d5586c34909be628fe5f1be8)
+++ b/src/test/java/src/algorithms/Validator.java	(date 1743008680021)
@@ -1,4 +1,4 @@
-package src;
+package src.algorithms;
 
 import java.lang.reflect.Field;
 import java.util.ArrayList;
Index: src/SortService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/SortService.java b/src/test/java/src/algorithms/SortService.java
rename from src/SortService.java
rename to src/test/java/src/algorithms/SortService.java
--- a/src/SortService.java	(revision 3aee43697d4282a5d5586c34909be628fe5f1be8)
+++ b/src/test/java/src/algorithms/SortService.java	(date 1743008679997)
@@ -1,11 +1,11 @@
-package src;
+package src.algorithms;
 
 import java.lang.reflect.Field;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 
-public class SortService implements Service{
+public class SortService implements Service {
     private List<Object> mainCollection = new ArrayList<>();
     private Validator validator = new Validator();
 
Index: src/Service.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Service.java b/src/test/java/src/algorithms/Service.java
rename from src/Service.java
rename to src/test/java/src/algorithms/Service.java
--- a/src/Service.java	(revision 3aee43697d4282a5d5586c34909be628fe5f1be8)
+++ b/src/test/java/src/algorithms/Service.java	(date 1743008680047)
@@ -1,4 +1,4 @@
-package src;
+package src.algorithms;
 
 public interface Service {
     public void execute();
Index: src/Program.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Program.java b/src/test/java/src/algorithms/Program.java
rename from src/Program.java
rename to src/test/java/src/algorithms/Program.java
--- a/src/Program.java	(revision 3aee43697d4282a5d5586c34909be628fe5f1be8)
+++ b/src/test/java/src/algorithms/Program.java	(date 1743008679964)
@@ -1,4 +1,4 @@
-package src;
+package src.algorithms;
 
 import java.util.ArrayList;
 import java.util.List;
Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/test/java/src/Main.java
rename from src/Main.java
rename to src/test/java/src/Main.java
--- a/src/Main.java	(revision 3aee43697d4282a5d5586c34909be628fe5f1be8)
+++ b/src/test/java/src/Main.java	(date 1743008679977)
@@ -1,5 +1,7 @@
 package src;
 
+import src.algorithms.Program;
+
 public class Main {
     public static void main(String args[]){
         Program program = new Program();
Index: src/ClassInspector.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/ClassInspector.java b/src/test/java/src/algorithms/ClassInspector.java
rename from src/ClassInspector.java
rename to src/test/java/src/algorithms/ClassInspector.java
--- a/src/ClassInspector.java	(revision 3aee43697d4282a5d5586c34909be628fe5f1be8)
+++ b/src/test/java/src/algorithms/ClassInspector.java	(date 1743008679950)
@@ -1,4 +1,4 @@
-package src;
+package src.algorithms;
 
 import src.customs.Bus;
 import src.customs.Student;
diff --git a/src/algorithms/MyBinarySearch.java b/src/test/java/src/algorithms/MyBinarySearch.java
rename from src/algorithms/MyBinarySearch.java
rename to src/test/java/src/algorithms/MyBinarySearch.java
diff --git a/src/algorithms/MyArrayList.java b/src/test/java/src/algorithms/MyArrayList.java
rename from src/algorithms/MyArrayList.java
rename to src/test/java/src/algorithms/MyArrayList.java
diff --git a/src/customs/User.java b/src/test/java/src/customs/User.java
rename from src/customs/User.java
rename to src/test/java/src/customs/User.java
diff --git a/src/customs/Student.java b/src/test/java/src/customs/Student.java
rename from src/customs/Student.java
rename to src/test/java/src/customs/Student.java
diff --git a/src/customs/Bus.java b/src/test/java/src/customs/Bus.java
rename from src/customs/Bus.java
rename to src/test/java/src/customs/Bus.java
diff --git a/src/algorithms/MySorting.java b/src/test/java/src/algorithms/MySorting.java
rename from src/algorithms/MySorting.java
rename to src/test/java/src/algorithms/MySorting.java
